tutorials/3d/procedural_geometry/arraymesh.rst

```diff
@@ -59,6 +59,11 @@ Under ``_ready()``, create a new Array.
   .. code-tab:: gdscript GDScript
 
     var arr = []
+    
+  .. code-tab:: csharp 
+  
+    var arr = new Godot.Collections.Array ();
```

Comment:
The spaces before the `()` should be removed, as it's not part of the C# style guide. Same for `new List<Vector3> ()` below.